plugins/azure/sqldatabases/dbDataMaskingEnabled.js

```diff
@@ -0,0 +1,71 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Set Dynamic Data Masking for SQL Databases',
+    category: 'SQL Databases',
+    domain: 'Databases',
+    description: 'Set up dynamic data masking to protect sensitive data exposure in SQL databases.',
```

Comment:
```suggestion
    description: 'Ensures dynamic data masking is enabeld for all SQL databases.',
```

description should explain what the plugin does and not what should be done. 
'Set up dynamic data masking to protect sensitive data exposure in SQL databases.' should be in recommended action